demo/test/unit/test_ainit_fmod.gd

```diff
@@ -3,5 +3,6 @@ extends "res://addons/gut/test.gd"
 func before_all():
 	# set up FMOD
 	Fmod.set_software_format(0, Fmod.FMOD_SPEAKERMODE_STEREO, 0)
+	Fmod.set_dsp_buffer_size(512, 4)
```

Comment:
I'll now set this always, as it is also good practice to define the buffer size explicitly before initialisation.